### DISCUSSION TITLE
Some possible improvements

### DISCUSSION THREAD

**Discussion by jeceey** [kctekn/obsidian-TagsRoutes#9] [Ideas]
Hi @kctekn, I really like your plugin. That's a really nice idea! The design is simple and really makes you want to get lost in your notes. Here are some recommendations to help you improve the plugin:

- Add a button to reset settings 
- Open in the current tab rather than in a split view 
- Add a nice icon for the tab as for the original graph view
- Add a timelapse as for the original graph view 
- Add the option in the settings to hide the full path of a file and display only the name of the .md file
- Turn the ‘TagsRoutes’ folder into a hidden folder, as most people have no use for editing it, but make it possible for those who do want to modify it or read the reports from the settings.
- Add a light theme to the plugin for the background and make it adaptable with the main theme of obsidian. (In the settings you could choose: dark theme, light theme, or obsidian's default theme.)
- Perhaps improve the design of the option box on the node view
- Also add management and modification of groups and add a filter, alway in the the option box on the node view
- Maybe add some keyboard shortcuts

Thank you for this plugin, it's a great project. If I have any other ideas, I'll get back to you. Keep up the good work, you doing well :)

**Comment by kctekn** [comment]
Thank you so much for your feedback and for enjoying the plugin! Just read this message. I'm thrilled to hear it enhances your note exploration. I truly appreciate the time you took to share these suggestions.

Regarding the reset settings button, you can find it at the bottom of the scene's settings box. As for your other ideas, they’re fantastic and bring up some points I hadn’t considered before. Some are already in progress, while others might take a bit more time to implement.

Thanks again for your input, and please feel free to share any more suggestions anytime!
